### ISSUE TITLE
[Devine Architect] [Float]

### ISSUE THREAD

**Issue by Mx772** [QuantumXenon/origins-plus-plus#16]
**Origin**
Devine Architect

**Power**
Float

**Notes**

### v1.5

v1.5 sets the 'Float' ability to `Save Hotbar Activator` which conflicts with Dimension Traveler:
![image](https://user-images.githubusercontent.com/9059161/166720726-45467c20-0be5-469e-8e9e-4a83f68e456b.png)
![image](https://user-images.githubusercontent.com/9059161/166720779-efd8b80d-8a95-4915-8b1a-a490771f65df.png)
![image](https://user-images.githubusercontent.com/9059161/166720827-a0a91751-d602-4c59-a665-6d2763798e6a.png)

### v1.4.1

v1.4.1 previously set it to 'secondary' which worked.
![image](https://user-images.githubusercontent.com/9059161/166720886-c3dab92f-9f3c-4de3-a761-1401e0d0f514.png)
![image](https://user-images.githubusercontent.com/9059161/166720914-98990b37-8bac-4d4e-a54f-bd4e920d18b4.png)
![image](https://user-images.githubusercontent.com/9059161/166721077-c19f8793-28a5-45c8-a99d-1da1b936d51a.png)

Likely due to:
https://github.com/QuantumXenon/origins-plus-plus/commit/8a55a8d8803b931cbb86a22da5f7fb3fb973046f
Specifically:
https://github.com/QuantumXenon/origins-plus-plus/blob/8a55a8d8803b931cbb86a22da5f7fb3fb973046f/data/origins-plus-plus/powers/devine-architect/float.json#L25-L28

Previously in `data/origins-plus-plus/powers/devine-architect/flight.json` it was:
```
	   "key":{
		  "key":"key.origins.secondary_active",
		  "continuous":true
	   },
```

### TL;DR:
data/origins-plus-plus/powers/devine-architect/float.json ->

```
        "key": "key.saveToolbarActivator",
```
Should be ->
```
        "key": "key.origins.secondary_active",
```

**Comment by QuantumXenon** [comment]
Thanks for the report! I have fixed that and will be included in the next release
